package.json

```diff
@@ -142,10 +141,10 @@
     "ts-node": "^2.0.0",
     "tslint": "~4.3.1",
     "typedoc": "^0.5.3",
-    "typescript": "2.0.10",
+    "typescript": "~2.1.5",
```

Comment:
In Angular 2.3.x with TS 2.1.x the NGC compiler left the `compiled` directory mostly empty and didn't generate the `ngfactory` files. Now with Angular 2.4.3 our AoT build works perfectly with TypeScript 2.1 and the tests are passing with flying colors.

While the [issue](https://github.com/angular/angular/issues/13715) technically hasn't been closed yet, the problem seems to have been fixed. My opinion is that it is safe to merge since the tests that failed before due to the issue are now passing.